src/components/sidebar/routePathView/LinkListView.tsx

```diff
@@ -0,0 +1,193 @@
+                                {this.getNodeName(routePathLink.startNode)}
+                            </div>
+
+                            <div className={s.tableItem}>
+                                {this.getNodeName(routePathLink.endNode)}
+                            </div>
+
+                            <div className={s.tableItem}>
+                                {this.getHastusId(routePathLink)}
+                            </div>
+
+                            <div className={s.tableItemSmall}>
+                                {routePathLink.startNodeType}
+                            </div>
+                        </div>
+                    </div>);
+            }
+            return;
+        });
+    }
+
+    public getHastusId = (routePathLink: IRoutePathLink) => {
+        return (routePathLink.startNode.stop && routePathLink.startNode.stop.hastusId) ?
+            routePathLink.startNode.stop.hastusId :
+            '-';
+    }
+
+    public getNodeName = (node: INode) => {
+        return (node.stop) ?
+            node.stop.nameFi :
+            '-';
+    }
+
+    public openLinkView = () => {
+        if (this.state.selectedRoutePathLink) {
+            this.props.sidebarStore!.setOpenLinkId(parseInt(this.state.selectedRoutePathLink, 10));
+            const linkViewLink = routeBuilder.to(subSites.link).toLink();
+            window.open(linkViewLink);
+        }
+    }
+
+    public getLinkViewButtonTitle = () => {
+        const routePathLinkId = this.state.selectedRoutePathLink;
+        if (routePathLinkId) {
+            return `Avaa linkki (id: ${this.state.selectedRoutePathLink})`;
+        }
+        return `Ei valittua linkkiä`;
+    }
+
+    public noRoutePathLinkSelected = () => {
+        if (this.state.selectedRoutePathLink) {
+            return false;
+        }
+        return true;
+    }
+
+    public setLinkTableFilter = (filter: string) => {
+        let filterType = 'P';
+        switch (filter) {
+        case filterTypes.STOPS: {
+            filterType = NodeType.STOP;
+            break;
+        }
+        case filterTypes.DISABLED: {
+            filterType = NodeType.DISABLED;
+            break;
+        }
+        case filterTypes.CROSSROAD: {
+            filterType = NodeType.CROSSROAD;
+            break;
+        }
+        case filterTypes.BORDER: {
+            filterType = NodeType.MUNICIPALITY_BORDER;
+            break;
+        }
+        case filterTypes.ALL: {
+            filterType = filterTypes.ALL;
+            break;
+        }
+        default: {
+            filterType = filterTypes.ALL;
+            break;
+        }
+        }
+        this.setState({
+            linkTableFilter: filterType,
+        });
+    }
+
+    public render(): any {
+        return (
+        <div className={s.linkListView}>
+            <div className={s.dropDownFilter}>
+                <Dropdown
+                    label={'Alkusolmutyyppi'}
+                    selected={'Pysäkki'}
```

Comment:
Strings in JSX can be written without {}
label='Alkusolmutyyppi'
selected='Pysäkki'

Reply:
fixd